assets/javascripts/discourse/models/chat-channel.js

```diff
@@ -115,6 +116,23 @@ const ChatChannel = RestModel.extend({
   },
 });
 
+ChatChannel.reopenClass({
+  create(args) {
+    args = args || {};
+    this._intiUserModels(args);
+    return this._super(args);
+  },
+
+  _intiUserModels(args) {
```

Comment:
This would probably need a unit test

Reply:
Not sure about this, frankly. In general, we should be testing _behavior_, not _implementation_. This just wraps user-like data into user models, which is clearly an implementation detail.

Reply:
Im fine with either here, and honestly tired of arguing on these subjects :)